packages/allure-codeceptjs/src/index.ts

```diff
@@ -208,11 +208,16 @@ class AllureReporter {
   }
 
   testAfter(test: CodeceptTest) {
+    const currentRetry = test._currentRetry;
```

Comment:
Looks awesome 🔥!!! Can I also ask you to add at lest one test for covering this kinda cases. If its too hard to implement, I can do it myself

Reply:
@vovsemenv added